source/magpie/group.d

```diff
@@ -22,6 +22,43 @@ struct Group(GrpRowType...)
     /// Index for group
     Index grpIndex;
 
+private:
+    int positionInGroup(int axis)(string[] index, int grpPos)
```

Comment:
the general connent about `ptrdiff_t` apples here too.

Reply:
Addressed in 6517b4a